functions/utils/regex.js

```diff
@@ -10,8 +10,18 @@ const alphanumericPunctuationRegex = (message) => {
   return !/^[\x20-\x7E]*$/.test(message);
 };
 
+const alphanumericPunctuationRegexWithNewLine = (message) => {
+  return !/^[\x20-\x7E\n]*$/.test(message);
+};
+
```

Comment:
Looks good, can we add to our unit tests to validate that this is working as expected

Reply:
sure